### ISSUE TITLE
Never ending game

### ISSUE THREAD

**Issue by Kawasofft** [davidaf3/csgo-tracker#3]
Hello,
I have a little problem with csgo-tracker. When my game crash or I just close him, and after, I reconnect the game I was playing, there is now 2 different matches in the "Matches" panel. And the first of the two will never disepear.
Is it possible to add a button on the match page to finish the game manually ?
Thank you for this tracker that works very well.

**Comment by davidaf3** [comment]
Hi!
I'm planning on adding two buttons: one to delete the match and another one to manually end a match in progress. I'm a bit busy right now but I'll do it when I have some spare time.
In the meantime, if you know a bit about SQlite, you can access the app's DB that is stored in `.csgo-tracker/stats.db` inide your user's directory and delete the match using SQL. The database schema can be found [here](https://github.com/davidaf3/csgo-tracker/blob/feb45c02346bab084c9dd47ccb52287dd1bb6aee/restapi/schema.sql).

**Comment by davidaf3** [comment]
Also, did you close and open the app again before reconnecting to the game?

**Comment by Kawasofft** [comment]
No. Good idea, I will try it in my next game.

**Comment by davidaf3** [comment]
Hmm. The thing is, if you don't close the app, a new match shouldn't be created when you rejoin.
Because the app keeps the current match in memory, if you don't close and reopen it before reconnecting to the game, the new info should be added to the correct match, instead of creating a new one.

**Comment by Kawasofft** [comment]
I restarted CS without closing the app but a game is still created. That's strange because my game destroyed itself at the end of the game. I don't know why, but it's okay.

**Comment by davidaf3** [comment]
Which game was destroyed, the one that was created when you first entered the match or the one that was created when you reconnected?

**Comment by Kawasofft** [comment]
Both

**Comment by davidaf3** [comment]
I have released a new version of the app: [v0.2.1](https://github.com/davidaf3/csgo-tracker/releases/tag/v0.2.1). I have added a button to stop a match in progress and another one to delete the match. And in this new version, matches won't be deleted unless you press the delete button.

So if you exit a match or your game crashes, and then you reconnect, the new info should be added to the correct match (provided you didn't force it to end or delete manually).

To update your app, you just have to download the new release and run the installer.
